homeassistant/components/powerwall/__init__.py

```diff
@@ -281,6 +286,9 @@ async def _fetch_powerwall_data(power_wall: Powerwall) -> PowerwallData:
         grid_services_active=grid_services_active,
         grid_status=grid_status,
         backup_reserve=backup_reserve,
+        # batteries=batteries,
+        # Don't know why mypy is angry with me.
```

Comment:
Seems like there's some limit to type checking over 6 items in an asyncio.gather call.   I don't understand why, but somehow related to [this code](https://github.com/python/typeshed/blob/main/stdlib/asyncio/tasks.pyi#L134) the gather call of 6 items works with mypy but over 6 will return errors in type checking.